grid-demo/src/app/core/advanced-row-selection/advanced-row-selection.component.html

```diff
@@ -1 +1,62 @@
-Core - Advanced Row Selection
+<cds-grid
+  *ngIf="
+    {
+      vms: vms | async,
+      selectedMap: selectedMap | async,
+      selectedCount: selectedCount | async,
+      pageIndex: pageIndex | async,
+      pageCount: pageCount | async,
+      pageSize: pageSize | async,
+      entirePageSelected: entirePageSelected | async
+    };
+    let data
+  "
+  aria-label="basic datagrid demo"
+  height="400"
+>
+  <cds-grid-column type="action">
+    <cds-checkbox (click)="stopPropagation($event)">
+      <input (click)="selectAll()" type="checkbox" [checked]="data.entirePageSelected" aria-label="Select All" />
```

Comment:
This check-all behavior can't be turned off in the angular grid and I wanted to make the two grids behave identically. So, this will make the core grid behave just like the angular grid.

Reply:
I think there is benefit in showing slightly different behavior if there is something you can do in Core that you can't do in NG, but the checkbox is probably not important enough for that.

Reply:
I'll merge for now, but we can pretty easily add indeterminate behavior if we want.